api/mongo-migrations/1647374289696-apd-schema.js

```diff
@@ -0,0 +1,76 @@
+const logger = require('../logger')('mongoose-migrate/migrate-apd-schema');
+
+/**
+ * Update the APD schema to more closely match the front end nav/page sections
+ */
+async function up () {
+  require('../models/apd'); // eslint-disable-line global-require
+  
+  // Grab all APDs
+  const apds = await this('APD').find({}).lean();
+  
+  const convertContractorResources = contractorResources =>
+    contractorResources.map(contractorResource => {
+      return {
+        ...contractorResource,
+        hourly: {
+          data: { ...contractorResource.hourly.data }
```

Comment:
now that we moved useHourly out, do we want to unnest this from data, so the the values are just directly under hourly?

Reply:
yep, I like that. ill update it here and in the other branch